PhysicsTools/PatAlgos/python/producersLayer1/jetProducer_cfi.py

```diff
@@ -38,15 +38,16 @@
     discriminatorSources = cms.VInputTag(
         cms.InputTag("pfJetBProbabilityBJetTags"),
         cms.InputTag("pfJetProbabilityBJetTags"),
-        cms.InputTag("pfTrackCountingHighPurBJetTags"),
+#        cms.InputTag("pfTrackCountingHighPurBJetTags"),
         cms.InputTag("pfTrackCountingHighEffBJetTags"),
         cms.InputTag("pfSimpleSecondaryVertexHighEffBJetTags"),
-        cms.InputTag("pfSimpleSecondaryVertexHighPurBJetTags"),
+        cms.InputTag("pfSimpleInclusiveSecondaryVertexHighEffBJetTags"),
```

Comment:
For now only remove taggers. `pfSimpleInclusiveSecondaryVertexHighEffBJetTags` should be added only once new RelVals become available and are propagated to the list of input files for PAT tests.